### ISSUE TITLE
[6.x] Incorrect some basic class stats (unquiped character)

### ISSUE THREAD

**Issue by StormBytePP** [TrinityCore/TrinityCore#15245]
I, on retail, had my character unequipped to write down the basic stats on paper so I could compare to what TC has on the same race + class combination.

The testing was a draenei shaman lvl 100, completelly naked (so the equipment don't add any stat/buffs)

While most of them are OK (and I will not put here the good ones), many of them are just wrong, seems that some race / class server side buffs are not being applied.

These are the stats that change on a Draenei Shaman lvl 100 (created in TC in lvl 1, and typing .level 99)

| STAT        	| RETAIL 	| TC    	|
|-------------	|--------	|-------	|
| Intellect   	| 1107   	| 1162  	|
| Haste       	| +5%    	| +0%   	|
| Mastery     	| 58.50% 	| 0.00% 	|
| Multistrike 	| 20.00% 	| 0.00% 	|
| Spell Power 	| 1107   	| 1162  	|
| Mana Regen  	| 3200   	| 1600  	|
| Dodge       	| 3.36%  	| 1.68% 	|
| Block       	| 3.00%  	|  5%     	|

Tested on rev 8ad51b5a6eab5400723b58dd3ce9986e4f824bb1

**Comment by carlos93** [comment]
Your official Shaman has some talent specialization learn? That haste and multistrike buffs looks like Elemental passive talents.

**Comment by StormBytePP** [comment]
Yes, but mine on TC also has elemental talent learnt, so I am comparong same with same

**Comment by Shauren** [comment]
This is a wrong approach - you should compare untalented with untalented. The way you did it (with specs) we have no way of knowing if its a spec passive that is broken or base stats are wrong.

**Comment by StormBytePP** [comment]
Ok, will do it when I arribe home

**Comment by StormBytePP** [comment]
Ok, it took me some time, but this time, I collected my 100 level character from retail, got them naked, untalented, and also made them forgot specialization and wrote down the stats.

Character created in TC are raised level 100 with .level command.

|       STAT      	| DRAENEI SHAMAN RETAIL 	| DRAENEI SHAMAN TC 	| HUMAN PRIEST RETAIL 	| HUMAN PRIEST TC 	| UNDEAD DK RETAIL (UNHOLY PRESENCE) 	| UNDEAD DK RETAIL (UNHOLY PRESENCE) 	|
|:---------------:	|:---------------------:	|:-----------------:	|:-------------------:	|:---------------:	|:----------------------------------:	|:----------------------------------:	|
| Health          	|         53400         	|                   	|        53400        	|                 	|                53400               	|                                    	|
| Mana            	|         32000         	|                   	|        160000       	|                 	|                 100                	|                                    	|
| Item Level      	|           0           	|                   	|          0          	|                 	|                  0                 	|                                    	|
| Movement Speed  	|          100%         	|                   	|         100%        	|                 	|                100%                	|                                    	|
|                 	|                       	|                   	|                     	|                 	|                                    	|                                    	|
| Strength        	|          692          	|      **628**      	|         843         	|                 	|                1454                	|                                    	|
| Agility         	|          1346         	|      **1282**     	|         1067        	|                 	|                1069                	|                                    	|
| Intellect       	|          1107         	|      **1043**     	|         1042        	|     **1094**    	|                 567                	|                                    	|
| Stamina         	|          890          	|                   	|         890         	|                 	|                 890                	|                                    	|
|                 	|                       	|                   	|                     	|                 	|                                    	|                                    	|
| Critical Strike 	|         5,00%         	|                   	|        5.00%        	|                 	|                5.00%               	|                                    	|
| Haste           	|         +5.00%        	|     **+0.00%**    	|        +0.00%       	|                 	|               +10.00%              	|             **+0.00%**             	|
| Mastery         	|         0.00%         	|                   	|        0.00%        	|                 	|                0.00%               	|                                    	|
| Spirit          	|          784          	|                   	|         782         	|                 	|                 645                	|                                    	|
| Bonus Armor     	|           0           	|                   	|          0          	|                 	|                  0                 	|                                    	|
| Multistrike     	|         0.00%         	|                   	|        0.00%        	|                 	|                0.00%               	|                                    	|
| Leech           	|         0.00%         	|                   	|        0.00%        	|                 	|                0.00%               	|                                    	|
| Versatility     	|         0.00%         	|                   	|        0.77%        	|    **0.01%**    	|                0.00%               	|                                    	|
| Avoidance       	|         0.00%         	|                   	|        0.00%        	|                 	|                0.00%               	|                                    	|
|                 	|                       	|                   	|                     	|                 	|                                    	|                                    	|
| Damage          	|        770-771        	|    **733-735**    	|         1-2         	|                 	|               831-832              	|                                    	|
| Attack Power    	|          1346         	|      **1282**     	|          0          	|                 	|                1454                	|                                    	|
| Attack Speed    	|          1.91         	|      **1.90**     	|          2          	|                 	|           1.82 / 9.09s *           	|          1.82 / **9.99s**          	|
|                 	|                       	|                   	|                     	|                 	|                                    	|                                    	|
| Spell Power     	|          1107         	|      **1043**     	|         1042        	|     **1094**    	|                 567                	|                **0**               	|
| Mana Regen      	|          640          	|      **1600**     	|         3200        	|     **8000**    	|                 N/A                	|                                    	|
|                 	|                       	|                   	|                     	|                 	|                                    	|                                    	|
| Armor           	|           0           	|                   	|          0          	|                 	|                  0                 	|                                    	|
| Dodge           	|         3.36%         	|     **1.68%**     	|        3.00%        	|    **3.18%**    	|                3.00%               	|              **3.66%**             	|
| Parry           	|         0.00%         	|                   	|        0.00%        	|                 	|                3.00%               	|              **5.00%**             	|
| Block           	|         3.00%         	|     **5.00%**     	|        0.00%        	|                 	|                0.00%               	|                                    	|

* Asterisk means rune speed
* Empty values are values which **match** retail
* Death knight has the unholy presence active, as it seems I can not have no presence (also presence buff itself seems bugged as you can see in Haste stat)

**Comment by StormBytePP** [comment]
@Shauren: [This is a wrong approach - you should compare untalented with untalented. The way you did it (with specs) we have no way of knowing if its a spec passive that is broken or base stats are wrong.]
This demonstrates that both, **basic stats**, **passive** and also **not passive** spec are broken:

* Base stats bad (as seen in the table above)
* Passive spec (as seen in Haste and Mastery of my first table of my shaman)
* Active spec (as seen in table above, in DK, about Haste that gives the unholy presence)

**Comment by StormBytePP** [comment]
Furthermore, can someone please test other classes? I don't have more lvl 100 to write here as I did so we can have a complete list of stats of all classes

**Comment by funjoker** [comment]
The stats were taken from bluepost beta data.
Release changed them also patches.
It's hard to maintane
